### PR TITLE
Update instruction for easy access to Humacs & Packet

### PR DESCRIPTION
**Pull request by Riaankl**
Please review and comment

### CODE REVIEW

File: instructions.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+## Steps
+1) get kind or minikube cluster up
+2) git clone https://github.com/humacs/humacs
+3) cd humacs/infra/cluster-api
+4) cp the secrect.env.example to make your own
+5) update the secrets.env you created in line 5 to give it a distinct  `CLUSTER_NAME`
+TODO: We need to generate a string to add to the name 
+7) run `./cluster-setup.sh`
+8) After the script hits  `[  OK  ] Reached target Cloud-init target.` type the following keys in sequence: "enter ~ ."
+9) we have to rebuild the apisnoop image to include that kubeconfig
```
Comment (BobyMCbobs) [inline instructions.md:72]: not sure what this refers to.
The APISnoop container image isn't rebuilt nor should it include a kubeconfig.

File: instructions.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+`~$ ssh-add -l`
+
+The agent has no identities.
+
+`~$ ssh-add`
+
+Identity added: /home/foo/.ssh/id_rsa (foo[@ii.coop](mailto:riaan@ii.coop))
+
+`~$ Kind Delete cluster` - To ensure you have no Kind instance running yet
```
Comment (BobyMCbobs) [inline instructions.md:109]: These commands should be lowercased: `kind delete cluster`

**Comment by BobyMCbobs** [comment]
Looks good!
Left a few comments.

@Riaankl, can this file be relocated into the docs folder?

**Comment by Riaankl** [comment]
Stale PR

**Pull request closed without merging**